### DISCUSSION TITLE
bug: cost usage are not getting captured automatically from llm response

### DISCUSSION THREAD

**Discussion by ajeet6261** [langfuse/langfuse#3305] [Support]
### Describe the bug

we have deployed langfuse in our private machine but for every conversation between agent there is no tracking of cost. Apart from cost rest of tracking are available.

### To reproduce

make llm api call and check langfuse UI dashboard it show latency and other information but cost tracking is 0 always.

### Are you interested to contribute a fix for this bug?

No

**Comment by marcklingen** [comment]
happy to help, can you share how you integrated with langfuse?

**Comment by marcklingen** [comment]
closing as I am missing information here
